<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_Search">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>500</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_starttime">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_endtime">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_searchpath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inputPath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_SearchResult">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>661</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_PhotoShow">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索结果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_key">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>关 键 字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_KeyWord">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_StartTime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_EndTime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>如：E:/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SsysShow">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>系统显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SoftShow">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>软件显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_FileShow">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>文件夹显示</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_SearchPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AddDefaultPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delDefaultPath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TempPath">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
